thermo/EquationsOfState.ipynb

```diff
@@ -0,0 +1,5059 @@
+    "    \n",
+    "    states = ct.SolutionArray(gas, len(p))\n",
+    "    X = 'CO2:1.0'\n",
+    "    states.TPX = T, p, X\n",
+    "    \n",
+    "    u = states.u\n",
+    "    h = states.h\n",
+    "    s = states.s\n",
+    "    cp = states.cp\n",
+    "    cv = states.cv\n",
+    "        \n",
+    "    # Get the relative enthalpy, entropy and int. energy with referance to the first point\n",
+    "    u = u - u[0]    \n",
+    "    s = s - s[0]    \n",
+    "    h = h - h[0]\n",
+    "        \n",
+    "    return  h, u, s, cp, cv\n",
+    "\n",
+    "def get_thermo_CoolProp(T, p) :\n",
+    "    \n",
+    "    n = len(p)\n",
+    "    u = np.zeros(n)\n",
+    "    h = np.zeros(n)\n",
+    "    s = np.zeros(n)\n",
+    "    cp = np.zeros(n)\n",
+    "    cv = np.zeros(n)\n",
+    "    \n",
+    "    for i in range(n):\n",
+    "        \n",
+    "        #PropsSI(\"T\",\"P\",101325,\"Q\",0,\"REFPROP::CO2\")\n",
+    "        \n",
+    "        u[i] = PropsSI(\"U\",\"P\", p[i],\"T\", T,\"HEOS::CO2\")     # kJ/kg/K\n",
+    "        h[i] = PropsSI(\"H\",\"P\", p[i],\"T\", T,\"HEOS::CO2\")\n",
+    "        s[i] = PropsSI(\"S\",\"P\", p[i],\"T\", T,\"HEOS::CO2\")\n",
+    "        cp[i] = PropsSI(\"C\",\"P\", p[i],\"T\", T,\"HEOS::CO2\")\n",
+    "        cv[i] = PropsSI(\"O\",\"P\", p[i],\"T\", T,\"HEOS::CO2\")\n",
+    "        \n",
+    "    # Get the relative enthalpy, entropy and int. energy with referance to the first point\n",
+    "    u = u - u[1]    \n",
+    "    s = s - s[1]    \n",
+    "    h = h - h[1]    \n",
+    "        \n",
+    "    return  h, u, s, cp, cv\n",
+    "\n",
+    "def plot(T, p, a_Ideal, a_RK, a_CoolProp, name) :\n",
+    "    \n",
+    "    line_width = 3\n",
+    "    n = len(p)\n",
+    "    \n",
+    "    plt.figure()\n",
+    "    plt.plot(p/1e5, a_Ideal,'-',color='b', linewidth = line_width)\n",
+    "    plt.plot(p/1e5, a_RK,'-',color='r', linewidth = line_width)\n",
+    "    plt.plot(p/1e5, a_CoolProp,'-',color='k', linewidth = line_width)\n",
+    "    plt.xlabel('Pressure [Bar]')\n",
+    "    plt.ylabel(name)\n",
+    "    plt.legend(['Ideal EoS','RK EoS', 'CoolProp'], prop={'size': 14}, frameon=False)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. EoS Comparison based on thermodynamic properties \n",
+    "\n",
+    "This is the main subroutine that compares and plots the thermodynamic values obtained using three equations of state. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Main function\n",
+    "\n",
+    "# Input parameters\n",
+    "T = 300                               # Temperature is constant [unit:K]\n",
+    "p = 1e5*np.linspace(1, 100, 1000)     # Pressure is varied from 1 to 100 bar [unit:Pa]\n",
+    "\n",
+    "# Read the ideal gas-phase\n",
+    "gasIdeal = ct.Solution('data/co2-thermo.yaml', 'CO2-Ideal', transport=None)\n",
```

Comment:
Corrected names to `gas_ideal` and `gas_RK`.  I did not use `ideal_gas `and `redlich_kwong `as variable names, mainly because that might be confusing with the actual phase-names in Cantera.

Reply:
But using `gas_RK` implies that RK can only represent has phases, which isn't correct. Maybe `redlich_kwong_phase`?

Reply:
That will do. I modified these names from `gas_RK `to `redlich_kwong_phase` and `gas_ideal `to `ideal_gas`.